agent/api/task/task.go

```diff
@@ -3682,7 +3685,8 @@ func (task *Task) ToHostResources() map[string]*ecs.Resource {
 func (task *Task) HasActiveContainers() bool {
 	for _, container := range task.Containers {
 		containerStatus := container.GetKnownStatus()
-		if containerStatus >= apicontainerstatus.ContainerPulled && containerStatus <= apicontainerstatus.ContainerResourcesProvisioned {
+		if containerStatus >= apicontainerstatus.ContainerManifestPulled &&
```

Comment:
nit: this if-statement seems like it would make more sense to be `containerStatus > ContainerStatusNone &&  containerStatus < ContainerStatusStopped`

Reply:
Yeah with current states both are equivalent but I get your point. It's a bit out of scope of this PR so I'd rather not change that logic here.